app/models/grda_warehouse/service_history_service_materialized.rb

```diff
@@ -32,7 +32,7 @@ def double_check_materialized_view(client_ids)
     return if incorrect_dates.blank?
```

Comment:
I can't comment on it, but up on line 11, do we still want `NotifierConfig`?  Also line 20 `setup_notifier`. I was thinking we'd just update the `ping` method to take whatever we needed for Sentry instead of adding `Sentry.capture` inline.  That way the `setup_notifier` calls could be used to tag things appropriately.  Maybe I'm thinking more about the logging updates than the crash report updates though.

If `ping` took a message and optional hash with the error and additional fields, then it could just log the message and send everything along to Sentry if there was an additional hash.

Reply:
Just pushed a commit trying out what I think you're describing, how does that look?

Reply:
That looks cleaner to me, it also allows us to easily transition to logging only instead of slack, and will allow for tagging with the "username" for aggregating the logs/generating metrics.  Thanks!